src/en/getting-started.md

```diff
@@ -1,255 +1,323 @@
 Title: Getting started with Juju
 
-# Getting started with Juju 
+# Getting started with Juju
 
-These instructions will get you up and running and deliver the best-possible
-experience with Juju. At the moment, that means using the latest release of
-Ubuntu with '[Long Term Support][long-term-support]' (LTS): [16.04 LTS (Xenial)][Xenial-download].
+Juju works across public clouds, private clouds, and locally with LXD based
+deployment and operating of applications. Juju works through a controller that
+will process all of the events that occur. These instructions will walk you
+through using the hosted Juju as a Service (JAAS) controller and requires you
+to have public cloud credentials ready. If you'd prefer to test it out
+locally, we've got a tutorial on setting up a [LXD based controller over
```

Comment:
'over here' sounds a little too casual to me, although I appreciate you're trying to be less formal. We'd normally write something more like `see our _Getting started with Juju and LXD_ guide further details`.